styles/styles.css

```diff
@@ -808,6 +816,100 @@ main .section.responsive-title h1 {
   font-size: 16px;
 }
 
+/* REDESIGN Buttons */
+.redesign-v2 a.button.primary,
+.redesign-v2 button.primary,
+.redesign-v2 a.button.secondary,
+.redesign-v2 button.secondary,
+.redesign-v2 a.button.tertiary,
+.redesign-v2 button.tertiary {
+  align-items: center;
+  border-radius: 2px;
+  border: 1px solid transparent;
+  display: inline-flex;
+  font-family: var(--ff-body);
+  font-size: var(--button-font-size);
+  font-weight: normal;
+  gap: 10px;
+  justify-content: center;
+  letter-spacing: 1.12px;
+  line-height: var(--button-line-height);
+  min-width: 80px;
+  padding: 13px 20px 9px;
+  text-align: center;
+  text-decoration: none;
+  transition: all var(--duration-small) var(--easing-standard);
```

Comment:
Transitions shouldn’t have `all`. Please replace with `background-color`

Reply:
applying to color, border-color and background-color now